com.unity.render-pipelines.universal/Runtime/Passes/CopyColorPass.cs

```diff
@@ -39,7 +41,21 @@ public CopyColorPass(RenderPassEvent evt, Material samplingMaterial, Material co
         /// </summary>
         /// <param name="source">Source Render Target</param>
         /// <param name="destination">Destination Render Target</param>
+        [Obsolete]
```

Comment:
It's because `com.unity.testing.urp/Scripts/Runtime/CameraCallbackTests.cs` uses this `Setup` function with `cameraColorTarget` which is `RenderTargetIdentifier` (I also forgot to set that as obsolete).

I left it in just to test that it would still work. Since `CopyColorPass` is in the internal namespace, maybe I should remove this?

Reply:
Ahh okay. Hmm, good question whether we should. Is there a plan about how we fully transition to RTHandle?

Reply:
First, adding compatibility with both. Our code should use RTHandles and RTI use should be marked as Obsolete/Deprecated. A following release of unity should remove the compatibility with RTI and our internal code should be cleaned up with all backwards compatible RTIs removed.

Reply:
That sounds good 👍